hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/fsck/TestContainerHealthTask.java

```diff
@@ -89,7 +92,7 @@ public void testRun() throws Exception {
 
     // Create 6 containers. The first 5 will have various unhealthy states
     // defined below. The container with ID=6 will be healthy.
-    List<ContainerInfo> mockContainers = getMockContainers(6);
+    List<ContainerInfo> mockContainers = getMockContainers(7);
```

Comment:
Need to revise the comment above as well.

Reply:
Done.